css/style.css

```diff
@@ -0,0 +1,504 @@
+}
+
+.header__cart {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.cart-icon {
+  margin-left: 5px;
+}
+
+.header__language-block {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  margin: 0;
+  padding: 0;
+}
+
+.header__language-image {
+  width: 20px;
+  height: 20px;
+}
+
+.promo {
+  background-image: url(../image/promo.png);
+  display: block;
+  width: 100%;
+  height: 281px;
+  margin-bottom: 53px;
+}
+
+.promo__block {
+  display: flex;
+  flex-direction: column;
+  width: 209px;
+  padding-left: 60px;
+  padding-top: 89px;
+  justify-content: center;
+  align-items: flex-start;
+}
+
+.promo__title {
+  font-size: 30px;
+  line-height: 29px;
+  display: flex;
+  align-items: center;
+  text-align: justify;
+  text-transform: uppercase;
+  color: #fff;
+  padding-bottom: 17px;
+}
+
+.promo__block-link {
+  display: flex;
+  justify-content: flex-start;
+}
+
+.link_white {
+  color: #fff;
+}
+
+.arrow {
+  padding-left: 7px;
+  padding-top: 1px;
+  width: 44px;
+  display: inline-block;
+}
+
+@media (max-width: 1169px) and (min-width: 1024px) {
+  .wrapper {
+    max-width: 930px;
+    margin: 0 auto;
+  }
+
+  .header {
+    grid-template-columns: 9.7% 51% 18.9% 8.8% 7.9% 3.7%;
+    margin-bottom: 45px;
+  }
+
+  .header__logo-image {
+    width: 61px;
+    height: 58px;
+  }
+
+  .menu {
+    margin-right: 140px;
+  }
+
+  .header__search {
+    width: 160px;
+  }
+
+  .search-input {
+    width: 161px;
+  }
+
+  .search-icon {
+    left: 146px;
+  }
+
+  .cart-icon {
+    margin-left: 4px;
+  }
+
+  .header__language-image {
+    width: 18px;
+    height: 19px;
+  }
+
+  .promo {
+    background-image: url(../image/promo-1024.png);
+    height: 254px;
+    margin-bottom: 68px;
+  }
+
+  .promo__block {
+    padding-left: 54px;
+    padding-top: 76px;
+  }
+
+  .promo__title {
+    padding-bottom: 10px;
+  }
+
+  .arrow {
+    padding-left: 12px;
+    padding-top: 2px;
+    width: 47px;
+  }
+}
+
+@media (max-width: 1023px) and (min-width: 768px) {
+  .header {
+    grid-template-columns: 1fr 1fr;
+    padding-bottom: 0;
+    margin-bottom: 25px;
+    height: 59px;
+  }
+
+  .header__logo-image,
+  .menu,
+  .header__user-block,
+  .header__search,
+  .header__cart,
+  .header__language-block {
+    display: none;
+  }
+
+  .header__logo-image_adaptive {
+    display: flex;
+    height: 34px;
+    width: 157px;
+    margin-left: 43px;
+    align-items: center;
+  }
+
+  .header__menu-bar {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+  }
+
+  .menu-bar {
+    padding-right: 44px;
+  }
+
+  .promo {
+    height: 415px;
+    margin-bottom: 67px;
+    background-size: cover;
+    background-position: 17%;
+  }
+
+  .promo__block {
+    padding-left: 44px;
+    padding-top: 121px;
+  }
+
+  .promo__title {
+    font-size: 50px;
+    line-height: 48px;
+    padding-bottom: 46.5px;
+  }
+
+  .link {
+    font-size: 20px;
+    line-height: 19px;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/chudik_css_end/pullRequest?prid=6014585)